source/features/convert-release-to-draft.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import LoadingIcon from '../github-helpers/icon-loading';
+import {getRepoURL} from '../github-helpers';
+
+async function convertToDraft({delegateTarget: draftButton}: delegate.Event<MouseEvent, HTMLButtonElement>): Promise<void> {
+	try {
+		await api.expectToken();
+		draftButton.textContent = 'Converting...';
+		draftButton.append(<LoadingIcon className="ml-2" width={16}/>);
+
+		const tagName = location.pathname.split('/').pop()!;
+		const {id: releaseID}: Record<string, number> = await api.v3(`repos/${getRepoURL()}/releases/tags/${tagName}`);
+		const response = await api.v3(`repos/${getRepoURL()}/releases/${releaseID}`, {
+			method: 'PATCH',
+			body: {
+				draft: true
+			}
+		});
+
+		location.pathname = [getRepoURL(), 'releases', 'edit', response.tag_name].join('/');
```

Comment:
GitHub url looks funny here since we are change the "branch". Use it anyway?

Reply:
> we are change the "branch"

Isn't that the whole reason that GitHubURL exists? It was fine.